### ISSUE TITLE
The max_key_len in data_utils is not defined. 212,213line.

### ISSUE THREAD

**Issue by xfysec** [cuhksz-nlp/ASA-TGCN#1]

**Comment by okjer** [comment]
try self.max_key_len->self.tokenizer.max_seq_len , the matrix should be phalanx. There are too many bugs in this code.

**Comment by xfysec** [comment]
> try self.max_key_len->self.tokenizer.max_seq_len , the matrix should be phalanx. There are too many bugs in this code.

OK, I got it, thanks a lot.

**Comment by okjer** [comment]
but after fix this bug, I can run run_sample.sh with F1 = 0.2222

**Comment by xfysec** [comment]
> but after fix this bug, I can run run_sample.sh with F1 = 0.2222

Are you train on the given limited data? I run asa_tgcn_main.py with val_acc and val_f1 all both 0.

**Comment by zfq-hit** [comment]
> Are you train on the given limited data? I run asa_tgcn_main.py with val_acc and val_f1 all both 0.
yes. At first you should google bert-large-uncased and download pytorch_model.bin,  config.json,  ,and vocab.txt .Second run the fitst command in run_sample.sh, since it will both do train and val. If you want to run the second command ,check where your model save and modify the path.

**Comment by yuki1111111** [comment]
> _No description provided._

您好，想问一下您跑通代码了吗，可否留个邮箱或者其他联系方式向您请教一下

**Comment by xzm123456** [comment]
too many bugs!
